packages/taquito-local-forging/src/michelson/codec.ts

```diff
@@ -97,6 +97,27 @@ export const extractRequiredLen = (value: Uint8ArrayConsumer, bytesLength = 4) =
   return value.consume(valueLen);
 };
 
+/**
+ *
```

Comment:
could you add a `@description` annotation just to outline what the function does